### DISCUSSION TITLE
2005/11/21/%E4%B8%BA-vc-%E5%AE%89%E8%A3%85-stlport

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#62] [Announcements]
# 2005/11/21/%E4%B8%BA-vc-%E5%AE%89%E8%A3%85-stlport

1. 从 www.stlport.com 下载。

https://blog.qizhen.xyz/2005/11/21/%E4%B8%BA-vc-%E5%AE%89%E8%A3%85-stlport

**Comment by ruanqizhen** [comment]
之前博客下的留言：

鑫
2007年01月11日 11:41 编辑
STL Port干什么用的呢

回复
Qizhen
2006年01月17日 10:03 编辑
STL Port 使用时的配置在 C:\\Program Files\\Microsoft Visual Studio\\VC98\\Include\\stlport\\stl_user_config.h （默认路径）中。比如，如果你的程序没有用到任何 IOSTREAMS，把宏定义“# define _STLP_NO_IOSTREAMS 1”打开，可以使你的程序不依赖于 STL Port 的任何 DLL 文件。

回复
